testsuite/tests/ui/conftest.py

```diff
@@ -152,26 +154,46 @@ def devel_login(account, custom_devel_login):
 
 # pylint: disable=unused-argument, too-many-arguments
 @pytest.fixture(scope="module")
-def custom_ui_tenant(master_login, navigator, threescale, testconfig, request, master_threescale):
+def custom_ui_tenant(master_login, navigator, threescale, testconfig, request, master_threescale, browser):
     """Parametrized custom Tenant created via UI"""
 
+    @backoff.on_predicate(backoff.constant, lambda ready: not ready, interval=6, max_tries=5, jitter=None,
+                          on_backoff=lambda _: browser.selenium.refresh())
+    def _backoff(func):
```

Comment:
change the name to something more meaningful and add documentation

Reply:
Is it better?